site/src/pages/WorkspacesPage/WorkspacesPageView.tsx

```diff
@@ -76,6 +84,78 @@ const WorkspaceHelpTooltip: React.FC = () => {
   )
 }
 
+const OutdatedHelpTooltip: React.FC<{ onUpdateVersion: () => void }> = ({ onUpdateVersion }) => {
```

Comment:
I think this file is getting really big.

Reply:
Yes, it is but, for now, I think it is ok.  What do you think?

Reply:
We can leave it for now. I want to talk about directory structure at the next FE Variety - maybe we can look at pulling stuff apart then.